anesthetic/utils.py

```diff
@@ -48,8 +48,8 @@ def quantile(a, q, w):
         w = numpy.ones_like(a)
     i = numpy.argsort(a)
     c = numpy.cumsum(w[i])
-    c /= c[-1]
-    icdf = interp1d(c/c[-1], a[i])
+    c /= c.values[-1]
+    icdf = interp1d(c/c.values[-1], a[i])
```

Comment:
Did you find that there was a bug that needed this fix?

Reply:
Doesn't work without this for me... Threw some error.

Reply:
Hopefully when I have finally got round to a proper test coverage this kind of issue would be thrown up anyway